crypto/schnorr.c

```diff
@@ -0,0 +1,78 @@
+#include "schnorr.h"
```

Comment:
Add copyright header (the same as in `schnorr.h`)

Reply:
done

Reply:
> Add copyright header (the same as in `schnorr.h`)

done